src/wh_12_1/OccurrenceCounter.java

```diff
@@ -0,0 +1,26 @@
+package wh_12_1;
+
+import java.util.List;
+import java.util.Map;
+
+public class OccurrenceCounter {
+    static Map<Integer, Integer> countOccurrence(List<Integer> numbersFromFile, Map<Integer, Integer> countOccurrence) {
+        int iterator = numbersFromFile.size();
+        for (int i = 0; i < iterator; i++) {
+            int numberToCheck = numbersFromFile.get(i);
+            countOccurrence.put(numberToCheck, numberOfOccurrence(numberToCheck, numbersFromFile));
+        }
+        return countOccurrence;
+    }
+
+    static int numberOfOccurrence(int numberToCheck, List<Integer> numbersFromFile) {
```

Comment:
zliczenuie wszystkich wystapien z listy to jest jeden sposob. Ale mozesz to zorbic be ztego : przy kazdej liczby mozesz sprawdzic jaka jest wartosc lementu  dla `numverTocheck` i dodac jeden do tej wartosci